TTBot/Program.cs

```diff
@@ -275,7 +275,16 @@ private void ScaffoldDatabase()
                                     AS
                                     SELECT *, (select count(*) from EventSignup where EventId = event.Id) as ParticipantCount
                                     FROM [Event]
-                                    ");
+                                    ");
+
+                try
+                {
+                    connection.Execute(@"ALTER TABLE Event

```

Comment:
Can you remove this and add as a script in the /scripts/ folder please?

Reply:
Done